prow/cluster/jobs/all-presets.yaml

```diff
@@ -133,6 +133,45 @@ presets:
       - key: secret
         path: id_rsa
         mode: 0600
+- labels: 
+    preset-use-api-private: "true"
+  volumeMounts:
+  - mountPath: /etc/gitconfig
+    name: gitconfig
+    readOnly: true
+  volumes:
+  - name: gitconfig
+    secret:
+      secretName: gitconfig
+  env:
+  - name: GOPRIVATE
+    value: istio.io/api
+- labels:
+    preset-use-go-control-plane-private: "true"
+  volumeMounts:
+  - mountPath: /etc/gitconfig
```

Comment:
I thought it had to be in $HOME/.gitconfig?

Reply:
$HOME/.gitconfig might be better. Pulled /etc/gitconfig from docs, but I don't actually see it in that location on my system.